src/test/kotlin/hr/foi/pop/backend/services/UserServiceTest.kt

```diff
@@ -90,4 +92,58 @@ class UserServiceTest {
         val userWithBadUsername = templateRequestBodyForTesting.copy(username = "bad")
         assertThrows<UserCheckException> { userService.registerUser(userWithBadUsername) }
     }
+
+    @Test
+    @Transactional
+    fun givenUserWithStatusActive_onDeactivate_shouldDeactivateUser() {
+        //given
+        val deactivateUserDefinition: ActivateUserDefinitions = ActivateUserDefinitions.DEACTIVATE
+        val userIdForUserWithActiveFlag: String = "1"
+
+        //when
+        val user: User = userService.activateOrDeactivateUser(userIdForUserWithActiveFlag, deactivateUserDefinition)
+
+        //assert
+        Assertions.assertEquals(false, user.isAccepted)
```

Comment:
Hm. Maybe it would be a tad bit better if the tested user would be the one directly from the repository than the one returned by the method. What if the method lies and returns a non-persisted user. For instance, someone forgets/removes a `userRepository.save(user)` call and the test still passes?